conan/tools/_compilers.py

```diff
@@ -302,7 +302,7 @@ def _cppstd_gcc(gcc_version, cppstd):
         v14 = "c++1y"
         vgnu14 = "gnu++1y"
 
-    if Version(gcc_version) >= "5.1":
+    if Version(gcc_version) == "5" or Version(gcc_version) >= "5.1":
```

Comment:
Why not ``Version(gcc_version) >= "5"``?

Reply:
It looked more didactic given that the issue is that the later introduced version without the minor caused the error. But ok. I changed it. (a gcc 5.0 never existed)